pkg/k8sutil/k8sutil.go

```diff
@@ -40,12 +44,19 @@ func GetWatchNamespace() (string, error) {
 	return ns, nil
 }
 
-// errNoNS indicates that a namespace could not be found for the current
+// ErrNoNamespace indicates that a namespace could not be found for the current
 // environment
 var ErrNoNamespace = fmt.Errorf("namespace not found for current environment")
 
+// ErrRunLocal indicates that the operator is set to run in local mode (this error
+// is returned by functions that only work on operators running in cluster mode)
+var ErrRunLocal = fmt.Errorf("operator run mode forced to local")
+
 // GetOperatorNamespace returns the namespace the operator should be running in.
 func GetOperatorNamespace() (string, error) {
+	if os.Getenv(ForceRunModeEnv) == "local" {
```

Comment:
Refactor this into:

```Go
func IsRunModeLocal() bool {
    return os.Getenv(ForceRunModeEnv) == "local"
}
```

Or at least have a `type RunModeType string` for "local".

Reply:
Did both because we declare the run mode outside of the k8sutil package as well